### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by bin0mial**
## The Changes Include:
- Adding Admin Panel using ActiveAdmin.
- Allowing admin to CRUD categories
- Allowing admin to view and index only users.

### CODE REVIEW

File: spec/factories/admin_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :admin_user do
+    
+  end
```
Comment (a-ali) [inline spec/factories/admin_users.rb]: Kindly add the required fields.

File: spec/models/admin_user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe AdminUser, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (a-ali) [inline spec/models/admin_user_spec.rb:12]: Please add the required specs, at least the presence validations.

File: spec/factories/admin_users.rb (hunk 1 above)
Comment (bin0mial) [reply spec/factories/admin_users.rb]: Done 5d7d43e04a7593423a1c70ff4a2c2f0118ae4b5d

File: spec/models/admin_user_spec.rb (hunk 2 above)
Comment (bin0mial) [reply spec/models/admin_user_spec.rb:12]: Done 5d7d43e04a7593423a1c70ff4a2c2f0118ae4b5d

**Review by a-ali — APPROVED** [review]

**Pull request merged**